File: APIflow/python/tutorials/simpleMsg.py
import sys, getopt

exec(open("../APIFlow.py").read())

email, password, apiKey = sys.argv[1], sys.argv[2], sys.argv[3]

sessionKey = auth(email, password, apiKey)

#we've already setup simple.bpmn20.xml on staging
processId = startProcessWithMsg(sessionKey, "simpleMsg")
process = getProcess(sessionKey, processId)

#complete tasks with empty json body
for t in process["TASKS"]:
	res = completeTask(sessionKey, t["ID"])
	print(res)

File: APIflow/js/tutorials/simpleMsg.js
var email = ""
var password = ""
var apiKey = ""

var sessionKey = auth(email, password, apiKey)
var processId = startProcessWithMsg(sessionKey, "simpleMsg")
var process = getProcess(sessionKey, processId)

for(var t = 0; t < process.tasks.length; t++){
	var res = completeTask(sessionKey, t.ID);
	console.log(res);
}